GLMakie/Project.toml

```diff
@@ -43,3 +43,9 @@ julia = "1"
 
 [sources.Makie]
 path = "../Makie"
+
+[extras]
+DynamicQuantities = "06fc5a27-2a28-4c7c-a15d-362465fb6821"
```

Comment:
I think this should go into ReferenceTests and not into each backend individually. We moved most test dependencies there, so that backends dont need to worry about it independently.

Reply:
Oh cool, thanks, just added!